src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/PropertyMetadata.php

```diff
@@ -47,13 +41,6 @@ public function isMandatory(): bool
 
     public function toJsonSchema(): ?array
     {
-        if (null === $this->value) {
-            return null;
-        }
-
-        return [
-            'name' => $this->name,
-            'const' => $this->value,
-        ];
+        return null;
```

Comment:
okay i dont know how this works tbh. but does this still have an effect if we return null here?

Reply:
The thing with JSON Schema is that if a field is required or not is not returned by the field itself, but by the schema it belongs to. We make use of this in the [`SchemaMetadata`](https://github.com/sulu/sulu/blob/d5d841b/src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/SchemaMetadata.php#L47)

Reply:
👍